Resources/input/fsh/profiles/GEM_ERP_PR_Task.fsh

```diff
@@ -34,16 +34,15 @@ Description: "This resource manages the ePrescription workflow"
 * for.identifier only $identifier-kvid-10 or $identifier-pkv
 * performerType.coding from GEM_ERP_VS_OrganizationType (required)
 * performerType.coding 1..
-* input ..* MS
+* input ..2 MS
```

Comment:
Hier habe ich Bauchschmerzen, warum das einschränken?
Wenn wir in Zukunft doch weitere Elemente an den Task hängen wollen, müssen wir auch hier ein BC einführen, anstatt nur weitere Elemente zu fordern.
Ebenso schränkt es die Nachnutzbarkeit des Profils für andere ein.

Reply:
Das ist zurückgebaut in den Zustand wie es vorher war, s. https://simplifier.net/packages/de.gematik.erezept-workflow.r4/1.2.3/files/2316617

Willst du es trotzdem offen lassen? Dann wäre das eine (unbegründete) Änderung von 1.2.3 -> 1.3.0

Reply:
Nein, lassen wir es so. Ich wollte nur verhindern, dass wir weiter einschränken. Unglücklich das das damals schon passiert ist.